UnityProject/Assets/Scripts/UI/Core/Radial/Radial.cs

```diff
@@ -84,6 +85,35 @@ public virtual int MaxShownItems
 
 		public int ShownItemsCount => Math.Min(TotalItemCount, MaxShownItems);
 
+		public V VerifyNonChildReference<V>(V component, string missingReference) where V : UnityEngine.Object
```

Comment:
These two look very useful. Couldn't we move them to somewhere more general?

Reply:
I was thinking of having them as extension methods but wasn't sure if they would be used outside of the radials. I could look into that tomorrow.

Reply:
Ok, let's merge this for now then